runtime/onert/backend/cpu/ops/TransposeLayer.cc

```diff
@@ -64,7 +65,7 @@ void TransposeLayer::transposeQuant8()
 }
 
 void TransposeLayer::configure(const IPortableTensor *input, IPortableTensor *output,
-                               const std::vector<int> &perm)
+                               const IPortableTensor *perm)
```

Comment:
I'd reorder them as input / perm / output (e.g. inputs go first, then output).

Reply:
Okay, I will reorder it.